code/citadel/toys/dildos.dm

```diff
@@ -17,30 +17,7 @@ obj/item/weapon/dildo
 	var/random_color 	= TRUE
 	var/random_size 	= FALSE
 	var/random_shape 	= FALSE
-	//these lists are used to generate random icons, stats, and names
-	var/list/possible_colors = list(//mostly neon colors
```

Comment:
You could have simply put these as var/static/list as well. since they're only used here